account_financial_report/report/templates/open_items.xml

```diff
@@ -179,6 +181,10 @@
                         <span t-field="line.amount_residual" t-options="{'widget': 'monetary', 'display_currency': res_company.currency_id}"/>
                     </div>
                     <t t-if="line.currency_id">
+                        <!--## currency_name-->
+                        <div class="act_as_cell amount">
```

Comment:
No need for the name, the monetary will add currency symbol

Reply:
I'd rather add foreign currency abbreviation because an scenario could have 2 kind of dollar US ones and Canadian. With this column you can differ them.